lib/Ed25519Signature2018.js

```diff
@@ -1,15 +1,24 @@
 /*!
  * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
  */
+import * as base58btc from 'base58-universal';
 import jsonld from 'jsonld';
 import {JwsLinkedDataSignature} from '@digitalbazaar/jws-linked-data-signature';
 import {
   Ed25519VerificationKey2018
 } from '@digitalbazaar/ed25519-verification-key-2018';
+import {
+  Ed25519VerificationKey2020
+} from '@digitalbazaar/ed25519-verification-key-2020';
 
-import suiteContext from 'ed25519-signature-2018-context';
+import ed25519Signature2018SuiteContext from 'ed25519-signature-2018-context';
+import ed25519Signature2020SuiteContext from 'ed25519-signature-2020-context';
 // 'https://w3id.org/security/suites/ed25519-2018/v1'
-const SUITE_CONTEXT_URL = suiteContext.constants.CONTEXT_URL;
+const SUITE_CONTEXT_URL = ed25519Signature2018SuiteContext.constants
+  .CONTEXT_URL;
+// 'https://w3id.org/security/suites/ed25519-2020/v1'
+const SUITE_CONTEXT_URL_2020 =
```

Comment:
The contexts are already exported by the suites themselves (as `Ed25519VerificationKey2020.SUITE_CONTEXT`, no need to import them separately).